test/local_sensitivity/sde_nondiag.jl

```diff
@@ -370,30 +369,30 @@ end
 
   @info res_sde_pa
 
-   res_sde_u0a, res_sde_pa = adjoint_sensitivities(sol,EulerHeun(),dg!,tarray
+  res_sde_u0a, res_sde_pa = adjoint_sensitivities(sol,EulerHeun(),dg!,tarray
       ,dt=dtmix,adaptive=false,sensealg=InterpolatingAdjoint(noisemixing=true))
 
-  @test isapprox(res_sde_u0a, res_sde_u0, rtol=1e-4)
-  @test isapprox(res_sde_pa, res_sde_p, rtol=1e-4)
+  @test_broken isapprox(res_sde_u0a, res_sde_u0, rtol=1e-5)
+  @test_broken isapprox(res_sde_pa, res_sde_p, rtol=1e-5)
 
   res_sde_u0a, res_sde_pa = adjoint_sensitivities(sol,EulerHeun(),dg!,tarray
       ,dt=dtmix,adaptive=false,sensealg=InterpolatingAdjoint(noisemixing=true, noise=DiffEqSensitivity.ZygoteNoise()))
 
-  @test_broken isapprox(res_sde_u0a, res_sde_u0, rtol=1e-4)
-  @test_broken isapprox(res_sde_pa, res_sde_p, rtol=1e-4)
+  @test_broken isapprox(res_sde_u0a, res_sde_u0, rtol=1e-5)
+  @test_broken isapprox(res_sde_pa, res_sde_p, rtol=1e-5)
 
 
   res_sde_u0a, res_sde_pa = adjoint_sensitivities(sol,EulerHeun(),dg!,tarray
     ,dt=dtmix,adaptive=false,sensealg=InterpolatingAdjoint(noise=false, noisemixing=true))
 
-  @test_broken isapprox(res_sde_u0a, res_sde_u0, rtol=1e-4)
-  @test_broken isapprox(res_sde_pa, res_sde_p, rtol=1e-4)
+  @test_broken isapprox(res_sde_u0a, res_sde_u0, rtol=1e-5)
```

Comment:
I'd rather have it default to NoiseGrid and have an option for other, and then just have a few tests on the NoiseWrapper form. Tests marked as broken will be bound to break in the future.

Reply:
I'll have to look tomorrow what goes wrong with the `noise=false` case. Maybe something in `_jacNoise ` needs to be adjusted -- but I don't see why `BacksolveAdjoint` should work while `InterpolatingAdjoint` doesn't.